doc/features/address-completion.md

```diff
@@ -0,0 +1,37 @@
+# Addresses
+
+[TOC]
+
+## Overview
+
+It is possible to provide an auto-complete and a validation for addresses.
+The Addresses are read from the Address model.
+
+Currently, address entries are swiss addresses imported from the Swiss Post.
```

Comment:
```suggestion
Currently, address entries are swiss addresses imported from Swiss Post.
```

Reply:
maybe leave a comment / link on where to get this token from Swiss Post? is it free?

Reply:
That is something I do not know and could not find any information about. I want to ask @TheWalkingLeek once I get a hold of him.